docs: fixed links,revised the docs of overview, quickstart and first 2 parts of tutorial (#3877)
refinedev/refine: documentation/docs/getting-started/overview.md

slug: /
---

## What is refine?

**refine** is a React-based framework that enables rapid development of web applications. It eliminates the repetitive tasks demanded by **CRUD** operations and provides industry standard solutions for critical parts of a project like **authentication**, **access control**, **routing**, **networking**, **state management** and **i18n**.

**refine** is *headless by design*, offering unlimited styling and customization options.

## What do you mean by "headless" ?

Instead of being a limited set of pre-styled components, **refine** is a collection of helper `hooks`, `components` and `providers`. They are all decoupled from your *UI* *components* and *business logic*, so they never keep you from customizing your *UI* or coding your own flow.

> **refine** works *seamlessly* with any custom design or UI framework you may favor. For convenience, it ships with ready-made integrations for [Ant Design System](https://ant.design/), [Material UI](https://mui.com/), [Mantine](https://mantine.dev/) and [Chakra UI](https://chakra-ui.com/).

Author: ```suggestion
refine also ships with ready-made integrations for [Ant Design System](https://ant.design/), [Material UI](https://mui.com/), [Mantine](https://mantine.dev/), and [Chakra UI](https://chakra-ui.com/) for convenience.

All in all, refine just works seamlessly with all custom design or UI framework.
```
This sentence was a slight eye-sore of sorts, this new version of solves that problem while making so that the first sentence is still very much on display.
